pydsa/tests/test_linked_list.py

```diff
@@ -20,41 +18,41 @@ def test_steps(self):
 
     # Test is_empty
     def step1(self):
-        self.assertTrue(self.test_list.is_empty())
+        assert self.test_list.is_empty() == True
 
     # Test insert_at_start
     def step2(self):
         self.test_list.insert_at_start(5)
         # print(self.test_list.toString())
-        self.assertTrue(self.test_list.head.get_value() == 5)
-        self.assertIsNone(self.test_list.head.get_next())
-        self.assertFalse(self.test_list.is_empty())
+        assert self.test_list.head.get_value() == 5
+        assert self.test_list.head.get_next() == None
+        assert self.test_list.is_empty() == False
 
     # Test insert_at_end
     def step3(self):
         self.test_list.insert_at_end(6)
         curr = self.test_list.head
         while curr.get_next():
             curr = curr.get_next()
-        self.assertEqual(curr.get_value(), 6)
-        self.assertIsNone(curr.get_next())
+        assert curr.get_value() == 6
+        assert curr.get_next() == None
 
     # Test find
     def step4(self):
         self.test_list.insert_at_start(4)
-        self.assertEqual(self.test_list.head, self.test_list.find(4))
+        assert self.test_list.head == self.test_list.find(4)
 
     # Test delete
     def step5(self):
-        self.assertEqual(self.test_list.find(4), self.test_list.head)
+        assert self.test_list.find(4) == self.test_list.head
         self.test_list.delete(4)
         self.assertRaises(ValueError, self.test_list.find, 4)
```

Comment:
@aktech 
I've made the changes required. Removed the imports and changed the way I assert the tests. However, how do you suggest I should change this particular line?
What I'm basically checking here is if 4 was successfully deleted from the list.
Also, could you tell me why you guys are preferring this method instead of assertTrue? Is there any down sides to it? Or is it because you have decided on that convention?